### PR TITLE
[Revisit v0.20.0][nits] Fix hacks roll_nodes.sh

### PR DESCRIPTION
**Pull request by kenju**
- https://github.com/cookpad/terraform-aws-eks/commit/5a1ff54a43b9581eb286bddc526109fdea0e939d
- https://github.com/cookpad/terraform-aws-eks/commit/c474fcbade4a5d7d5d041ac3b6bc04acbda95794

### CODE REVIEW

File: hack/roll_nodes.sh [hunk 1]
```diff
@@ -72,7 +72,7 @@ done
 for NODE in $NODES_TO_ROLL
 do
   detach $NODE
-  kubectl drain --delete-local-data --ignore-daemonsets $NODE
+  kubectl drain --delete-emptydir-data --ignore-daemonsets $NODE
```
Comment (demijonathon) [inline hack/roll_nodes.sh:75]: I can't find this as a flag. The only option I've seen is --delete-local-data=false ?
https://github.com/kubernetes/kubernetes/issues/80228
Can you explain what this is doing (in the PR).

Comment (kenju) [reply hack/roll_nodes.sh:75]: There is one PR linked in that issue

https://github.com/kubernetes/kubernetes/pull/95076

Basically, only the name of this flag is changed. What it does is unchanged. The original flag (`--delete-local-data`) seems to be misleading, that's why the new flag `--delete-emptydir-data` was introduced.

**Review by demijonathon — APPROVED** [review]
LGTM

File: hack/roll_nodes.sh (hunk 1 above)
Comment (ettiee) [reply hack/roll_nodes.sh:75]: That PR is targeting the 0.20.0 release of kubernetes, should we introduce this change with our 0.20.0 release?

Comment (kenju) [reply hack/roll_nodes.sh:75]: > That PR is targeting the 0.20.0 release of kubernetes, should we introduce this change with our 0.20.0 release?

Right, maybe it's better to align the version with our release version.

So shall we revisit this PR when we're upgrading to v0.20.0 then?

Comment (ettiee) [reply hack/roll_nodes.sh:75]: Yes sounds good to me, I think it's cleaner to align client with server versions

**Review by errm — APPROVED** [review]
LGTM for 0.20.0

**Comment by kenju** [comment]
:point_right: #228

**Pull request closed without merging**